react/src/components/Nav.js

```diff
@@ -9,82 +7,72 @@ import { resetCart, addProduct, setProducts } from '../actions';
 
 import EPlogo from '../assets/empowerplant-logo.svg';
 
-class Nav extends Component {
-  static contextType = Context;
+function Nav(props) {
```

Comment:
Instead of `props`, can we pass the specific props used? At a glance looks like, `cart`, `frontendSlowdown` (I skimmed, but maybe others?)

Reply:
Confirmed that Nav.js was the only one that was missed. Resolved with https://github.com/sentry-demos/empower/pull/374/commits/da23022972a05785fb7ea831cc91f1101ce503ad